examples/mnist-inference-web/src/model.rs

```diff
@@ -0,0 +1,104 @@
+#![allow(clippy::new_without_default)]
+
+// Orginally copied from the burn/examples/mnist package
+
+use alloc::{format, vec::Vec};
+
+use burn::{
+    module::{Module, Param},
+    nn::{self, Initializer},
+    tensor::{backend::Backend, Tensor},
+};
+
+#[derive(Module, Debug)]
+pub struct Model<B: Backend> {
+    conv1: Param<ConvBlock<B>>,
+    conv2: Param<ConvBlock<B>>,
+    conv3: Param<ConvBlock<B>>,
+    // dropout: nn::Dropout,
+    fc1: Param<nn::Linear<B>>,
+    fc2: Param<nn::Linear<B>>,
+    activation: nn::GELU,
+}
+
+const NUM_CLASSES: usize = 10;
+
+impl<B: Backend> Model<B> {
+    pub fn new() -> Self {
+        let conv1 = ConvBlock::new([1, 8], [3, 3]); // out: [Batch,1,26,26]
+        let conv2 = ConvBlock::new([8, 16], [3, 3]); // out: [Batch,1,24x24]
+        let conv3 = ConvBlock::new([16, 24], [3, 3]); // out: [Batch,1,22x22]
+
+        let fc1 = nn::Linear::new(
+            &nn::LinearConfig::new(24 * 22 * 22, 32)
+                .with_bias(false)
+                // Initialize with zeros (by default is random)
+                .with_initializer(Initializer::Zeros),
+        );
+
+        let fc2 = nn::Linear::new(
+            &nn::LinearConfig::new(32, NUM_CLASSES)
+                .with_bias(false)
+                // Initialize with zeros (by default is random)
+                .with_initializer(Initializer::Zeros),
```

Comment:
I'm not sure we need to initialize the weights by zeros, could you check if this impacts performance?

Reply:
I am initializing with Zero so that the random function is not activated. The impact is not noticeable.

I can put the defaults back and come back with the data load optimization you suggested with the Initiate trait. This way the users will not be confused.